pkgs/development/python-modules/virt-firmware/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, cryptography
+, pefile
+}:
+
+buildPythonPackage rec {
+  pname = "virt-firmware";
+  version = "23.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "a2b4c876e4b82951f89d349170e9e7dbe4ea7b277838c9e7376f802b9a75c6d3";
```

```suggestion
    hash = "sha256-orTIduS4KVH4nTSRcOnn2+Tqeyd4OMnnN2+AK5p1xtM=";
```
Could you please use SRI hash instead?
https://nixos.org/manual/nixpkgs/stable/#fetchurl